scripts/quests/windurst/Making_Amends.lua

```diff
@@ -0,0 +1,94 @@
+-----------------------------------
+-- Making Amends
+-----------------------------------
+-- !addquest 2 3
+-- Hakkuru-Rinkuru : !pos -111 -4 101 240
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.MAKING_AMENDS)
+
+quest.reward =
+{
+    fame     = 150,
+    fameArea = xi.quest.fame_area.WINDURST,
+    title    = xi.title.QUICK_FIXER,
+    gil      = 1500,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.WINDURST) >= 2
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Hakkuru-Rinkuru'] = quest:progressEvent(274, 0, xi.items.ANIMAL_GLUE),
+
+            onEventFinish =
+            {
+                [274] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Hakkuru-Rinkuru'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.ANIMAL_GLUE) then
+                        return quest:event(277, 1500)
```

Comment:
What's the second value of 1500?
```suggestion
                        return quest:progressEvent(277, 1500)
```

Reply:
This is the reward amount. Some of the Windurst quests have gil rewards built into the dialog.
If I leave this out, it will say `Obtained  gil.`. Not sure there's a way to remove the duplicate dialog from the quest system reward? Anyway, I should be able to replace this magic number with `quest.reward.gil`.

Reply:
Yeah that would be great!

Reply:
Yeah that would be great!

Reply:
Yeah ok, it's done and I tested it works.